src/Event.cs

```diff
@@ -3,7 +3,27 @@
 
 namespace SpacetimeDB
 {
-    public interface IEventContext { }
+    public interface IEventContext
+    {
+
+    }
+
+    public interface IReducerEventContext
+    {
+
+    }
+
+    public interface ISubscriptionEventContext
+    {
+
+    }
+
+    public interface IErrorContext
+    {
+        public Exception Error { get; }
```

Comment:
Can we remove this? The error should already be in `ctx.Event`, and also exposed as a callback arg.

Reply:
I guess I can just call it Event

Reply:
All of these types (except `SubscriptionEventContext`) should have an `Event` field, right?

Reply:
It's there in the generated code. This interface pattern is harebrained but I don't want to change it rn